src/TenantCloud/APIVersioning/Constraint/Operator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TenantCloud\APIVersioning\Constraint;
+
+enum Operator: string
+{
+	case LESS = '<';
+	case LT = 'lt';
```

Comment:
Why so many operators? `lt` is `<`, keep just one. Same goes for `==` and `=` - they are the same operator.

Keep only `<`, `>`, `<=`, `>=`, `==` and `!=`, remove the rest